pkgs/applications/networking/instant-messengers/signal-desktop/default.nix

```diff
@@ -1,6 +1,9 @@
 { hostPlatform, callPackage }: {
-  signal-desktop = if hostPlatform.system == "aarch64-linux"
-    then callPackage ./signal-desktop-aarch64.nix { }
-    else callPackage ./signal-desktop.nix { };
+  signal-desktop = {
+    aarch64-linux = callPackage ./signal-desktop-aarch64.nix { };
+    x86_64-linux = callPackage ./signal-desktop.nix { };
+    aarch64-darwin = callPackage ./signal-desktop-aarch64-darwin.nix { };
+    x86_64-darwin = callPackage ./signal-desktop-darwin.nix { };
+  }.${hostPlatform.system};
```

Comment:
Shouldn't this be `targetPlatform`?

Reply:
The package originally used `hostPlatform`, so I just copied that into my PR. If `targetPlatform` is more favorable I can use that instead.

Reply:
Yeah, please change it to use the target platform. You only use host when you need to use something that runs on the host platform (example: compilers, linkers, build tools, etc.)